app/src/main/java/com/github/pockethub/ui/code/RepositoryCodeFragment.java

```diff
@@ -140,7 +140,7 @@ private void showLoading(final boolean loading) {
 
     private void refreshTree(final GitReference reference) {
         showLoading(true);
-        new RefreshTreeTask(repository, reference, getActivity()) {
+        new RefreshTreeTask(repository, reference, getContext().getApplicationContext()) {
```

Comment:
@StefMa I suppose this can not fix the memory leak. Ref [this post](http://www.androiddesignpatterns.com/2013/04/activitys-threads-memory-leaks.html), the `RefreshTreeTask` is a thread or contains a thread. This anonymous inner thread instance would hold a strong ref to the fragment which hold a strong ref to its activity, so the memory leak still exists.